client/components/Tracker/List/List.jsx

```diff
@@ -5,30 +5,25 @@ import styled from 'styled-components';
 import dummyData from './dummy_data';
 import ListItem from './ListItem';
 
-const List = ({ thoughtsList, setThoughtsList, getList }) => {
-  const [renderAmount, setRenderAmount] = useState(20);
+const List = ({ thoughtsList, getList, renderCount, setRenderCount }) => {
 
-  const [renderCount, setRenderCount] = useState(4);
 
   useEffect(() => {
-    getList();
-  }, []);
+    getList(renderCount);
+  }, [renderCount]);
 
   const setRenderCountHandler = () => {
-    if (renderCount <= thoughtsList.length) {
-      setRenderCount((prevRenderCount) => prevRenderCount + 4);
-    } else if (renderCount >= thoughtsList.length) {
-      setRenderAmount((prevRenderAmount) => prevRenderAmount + 20);
-      setThoughtsList(thoughtsList.concat(dummyData.slice(renderAmount, renderAmount + 20)));
-      setRenderCount((prevRenderCount) => prevRenderCount + 4);
-    }
+    setRenderCount(renderCount + 4);
   };
 
   return (
     <>
       <h2>List</h2>
-      {thoughtsList.filter((item) => thoughtsList.indexOf(item) < renderCount).map((item) => <ListItem item={item} />)}
-      <button onClick={() => setRenderCountHandler()}>Load More</button>
+      {thoughtsList.filter((item) => thoughtsList.indexOf(item) < renderCount - 1).map((item) => <ListItem item={item} />)}
+      {
+        renderCount > thoughtsList.length ? null
+          : <button onClick={() => setRenderCountHandler()}>Load More</button>
```

Comment:
We can remove the anonymous function in the onClick handler.

Reply:
@jessicaawhy, removed the anonymous function and committed it!